go.mod

```diff
@@ -26,7 +26,7 @@ require (
 	github.com/aws/smithy-go v1.22.3
 	github.com/bmizerany/pat v0.0.0-20210406213842-e4b6760bdd6f
 	github.com/canonical/go-dqlite/v3 v3.0.3
-	github.com/canonical/lxd v0.0.0-20251117151656-8e01bcf3d963
```

Comment:
All these LXD provider changes looks like we're using the wrong version of the LXD client. This was an oversight in 3.6 and now we're paying for it in 4.0 esp. because there are know breaking changes in the 6.6 that might affect us:
 https://documentation.ubuntu.com/lxd/latest/reference/release-notes/release-notes-6.6/#backwards-incompatible-changes

I'm not sure what to do here.